app/models/concerns/clone/intervention.rb

```diff
@@ -3,6 +3,8 @@
 class Clone::Intervention < Clone::Base
   def execute
     outcome.status = :draft
+    outcome.reports_deleted = false
+    outcome.data_cleared = false
```

Comment:
I we have default value we can skip this part

Reply:
`outcome.status = :draft` also is set here even though `:draft` is the default value. I believe, looking at `Clone::Base`, these flags would be simply copied from the source intervention